shoop/core/models/_service_behavior.py

```diff
@@ -168,3 +172,39 @@ def get_unavailability_reasons(self, service, source):
         groups_to_match = set(self.groups.all().values_list("pk", flat=True))
         if not bool(customer_groups & groups_to_match):
             yield ValidationError(_("Service is not available for any of the customers groups."))
+
+
+class RoundingBehavior(Enum):
+    ROUND_UP = 1
+    ROUND_DOWN = 2
+
+    class Labels:
+        ROUND_UP = _("round up")
+        ROUND_DOWN = _("round down")
```

Comment:
Provide also ROUND_HALF_UP and ROUND_HALF_DOWN and use the values of `decimal.ROUND_*` constants directly to avoid mapping those. They are strings , so the EnumIntegerField below has to be changed to EnumField too. Python's decimal module calls this rounding mode and IMHO that's better name than rounding behavior.

```python
class RoundingMode(Enum):
    ROUND_HALF_UP = decimal.ROUND_HALF_UP
    ROUND_HALF_DOWN = decimal.ROUND_HALF_DOWN
    ROUND_UP = decimal.ROUND_UP
    ...
    class Labels:
        ROUND_HALF_UP = _("round to nearest with ties going away from zero")
        ...
```

I don't think using those labels directly for the description of the cost is a good idea though...